src/assets/config/config.dev.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
Do not commit local config files into the code. Use `config.local.json` to override config for the development environment as mentioned in the README.